web/themes/custom/pineapple/pineapple.theme

```diff
@@ -33,6 +33,22 @@ function pineapple_theme_suggestions_field_alter(array &$suggestions, array $var
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_input_alter().
+ */
+function pineapple_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  $element = $variables['element'];
+  if (!empty($element['#name'])) {
+    $suggestions[] = 'input_' .$element['#name'];
+  }
+  if (!empty($element['#webform_id']) && !empty($element['#name'])) {
+    $suggestions[] = 'input_' . $element['#webform_id'] . '_' . $element['#name'];
+  }
+  if (!empty($element['#webform_id']) && !empty($element['#type'])) {
+    $suggestions[] = 'input_' . $element['#webform_id'] . '_' . $element['#type'];
```

Comment:
Add double underscores and put webform as opposed to the middle

Reply:
Sorry, I don't understand.

Reply:
Done